src/plugins/vis_default_editor/public/components/controls/filter.tsx

```diff
@@ -110,6 +110,7 @@ function FilterRow({
           dataTestSubj={dataTestSubj}
           bubbleSubmitEvent={true}
           languageSwitcherPopoverAnchorPosition="leftDown"
+          className="kbnTypeahead--small"
```

Comment:
I would add this as via a prop on `QueryStringInput` like `size: 's' | 'l'`, that way other implementations can reuse this as well without needing to remember the class name.